examples/manifests/consumer.yaml

```diff
@@ -15,15 +15,8 @@ spec:
       labels:
         app: consumer
     spec:
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-              - matchExpressions:
-                  - key: app
-                    operator: In
-                    values:
-                      - local
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
```

Comment:
This might not work in CI. There are multiple worker nodes in openshift aws CI environment. In hw-event-proxy e2e I kept the above affinity code and marked the first `Ready` worker node as `app=local`.
```
# label the first Ready worker node as local
label-node:
	@kubectl label --overwrite node $(shell kubectl get nodes -l node-role.kubernetes.io/worker="" | grep Ready | cut -f1 -d" " | head -1) app=local
```

Reply:
What I am doing here is not trying to land all under the same worker node. for CI it doesn't matter.

Reply:
Alright added affinity back